packages/core/src/deployment.ts

```diff
@@ -73,6 +95,6 @@ export class TransactionMinedTimeout extends Error {
 
 export class InvalidDeployment extends Error {
   constructor(readonly deployment: Deployment) {
-    super(`Invalid deployment ${deployment.txHash}`);
+    super(`Invalid deployment with address ${deployment.address} and txHash ${deployment.txHash}`);
```

Comment:
```suggestion
    super(`Invalid deployment with address ${deployment.address} and txHash ${deployment.txHash ?? 'unknown'}`);
```

Does this trigger the line length limit...?